docs/installing-solidity.rst

```diff
@@ -179,12 +189,98 @@ Install it using ``brew``:
     # eg. Install 0.4.8
     brew install solidity.rb
 
-Gentoo Linux has an `Ethereum overlay <https://overlays.gentoo.org/#ethereum>`_ that contains a solidity package.
-After the overlay is setup, ``solc`` can be installed in x86_64 architectures by:
+Static Binaries
+===============
 
-.. code-block:: bash
+We maintain a repository containing static builds of past and current compiler versions for all
+supported platforms at `solc-bin`_. It is also the location where you can find the nightly builds.
+
+The repository is not only a quick and easy way for end users to get binaries ready to be used
+out-of-the-box but it is also meant to be friendly to third-party tools:
+
+- The content is mirrored to https://solc-bin.ethereum.org where it can be easily downloaded over
```

Comment:
I still suggest we should go with something like binaries.soliditylang.org. As you mentioned @cameel, it would be possible just to have both URLs on S3, and both would work, so no need to add redirects either.

Any opinions @chriseth @franzihei ?

Reply:
Would it mean that we would mirror the content and have it "double" on both sites? I think having something on our domain in addition to the ethereum.org domain is always a good idea generally.

Reply:
It wouldn't even be mirrored. It would be the same exact Amazon S3 bucket behind both domains. They would be just alternate names for the same thing.

The only complication is that we might need assistance from devops team to generate the right SSL certificate since they control solc-bin.ethereum.org which is currently attached to the bucket. It must be a single SSL certificate valid for both and this might require re-verification with both domains.

Other that that, I see no technical problems. Just maybe that changing the domain again might make people confused about which one is deprecated. But it's already confusing so maybe doing it now is better than having to go through this later again.

Reply:
> But it's already confusing so maybe doing it now is better than having to go through this later again.

Exactly, if our goal is to migrate everything under `soliditylang.org`, this is a good opportunity to do so. We're not going to shut down `solc-bin.ethereum.org` anyhow, but if we are pinging people to update then we should prefer the one with soliditylang.org

Reply:
Done. It's now binaries.soliditylang.org in the text.